src/program/snabbvmx/doc/README.troubleshooting.md

```diff
@@ -550,24 +594,22 @@ softwires {
   { ipv4=10.10.0.0, psid=4, b4=2a02:587:f710::43 }
 }
 
-lab@ubuntu1:~/latest-snabb-binary/snabb/src/program/snabbvmx/tests/end-to-end/data$ tcpdump -r icmp-ipv4-in.pcap -e
-reading from file icmp-ipv4-in.pcap, link-type EN10MB (Ethernet)
-11:27:10.976929 90:e2:ba:94:2a:bc (oui Unknown) > 02:cf:69:15:81:01 (oui Unknown), ethertype IPv4 (0x0800), length 42: 10.0.1.100.domain > 10.10.0.0.1024: [|domain]
-11:27:10.988846 90:e2:ba:94:2a:bc (oui Unknown) > 02:cf:69:15:81:01 (oui Unknown), ethertype IPv4 (0x0800), length 42: 10.0.1.100.domain > 10.10.0.0.domain: [|domain]
-11:27:11.001278 90:e2:ba:94:2a:bc (oui Unknown) > 02:cf:69:15:81:01 (oui Unknown), ethertype IPv4 (0x0800), length 242: 10.0.1.100 > 10.10.0.0: ICMP echo reply, id 1024, seq 0, length 208
-11:27:11.017448 90:e2:ba:94:2a:bc (oui Unknown) > 02:cf:69:15:81:01 (oui Unknown), ethertype IPv4 (0x0800), length 242: 10.0.1.100 > 10.10.0.0: ICMP echo reply, id 53, seq 0, length 208
-11:27:11.029226 90:e2:ba:94:2a:bc (oui Unknown) > 02:cf:69:15:81:01 (oui Unknown), ethertype IPv4 (0x0800), length 242: 192.168.0.0 > 192.168.1.100: ICMP echo reply, id 1024, seq 0, length 208
-11:27:11.040811 90:e2:ba:94:2a:bc (oui Unknown) > 02:cf:69:15:81:01 (oui Unknown), ethertype IPv4 (0x0800), length 42: 192.168.0.0.domain > 192.168.1.100.1024: [|domain]
+lab@cgrafubuntu2:~$ tcpdump -n -r ipv4-in.pcap
+reading from file ipv4-in.pcap, link-type EN10MB (Ethernet)
+11:26:07.168372 IP 10.0.1.100.53 > 10.10.0.0.1024: [|domain]
 ```
 
 
 
-**running the interactive check**
+**running the interactive check**  
```

Comment:
Not an actual change, that's why we try not to commit trailing white spaces.

To prevent committing trailing white spaces, a good trick it to activate git's pre-commit hook. In your snabb repo, enter .git/hooks folder and rename pre-commit.sample to pre-commit.

```bash
snabb $ cd .git/hooks
snabb $ mv pre-commit.sample pre-commit
```